src/main/java/org/opentripplanner/util/OTPFeature.java

```diff
@@ -19,7 +19,7 @@ public enum OTPFeature {
   APIGraphInspectorTile(true),
   APIUpdaterStatus(true),
   MinimumTransferTimeIsDefinitive(false),
-  OptimizeTransfers(true),
```

Comment:
I think you want to revert these changes.

Reply:
True, I should not commit any changes to this class. It is reverted now.